airgun/entities/puppet_class.py

```diff
@@ -0,0 +1,58 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.puppet_class import PuppetClassDetailsView, PuppetClassesView
+
+
+class PuppetClassEntity(BaseEntity):
+
+    def search(self, value):
+        """Search for puppet class entity"""
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def read(self, entity_name):
+        """Read puppet class entity values"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.read()
+
+    def update(self, entity_name, values):
+        """Update puppet class values"""
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        view.fill(values)
+        view.submit.click()
```

Comment:
Let's not increase a scope of #119 and make a rule to add all necessary flash notifications checks for all new entities, what do you think? :)